### DISCUSSION TITLE
Praise and thank you thread :heart:

### DISCUSSION THREAD

**Discussion by tansangle** [pdfarranger/pdfarranger#843] [General]
Dear Developer Team,

As uptonow, this is the only way that I found so say thank to you, create a discussion topic :D

I wanted to give a big thank you for your work on pdfarranger. It has been an perfect tool for me in my work, and has saved me a lot of time. I appreciate all of the hard work that you and your team have put into it.

Thank you again for all that you do!

Sincerely, Sang Le

**Comment by LayersOfAbstraction** [comment]
Hi devs,

I am giving my thanks without creating more discussions. This helped me get good use out of open-source resume creators and trimmed off pages I didn't want.

Kind regards,
Jordan Nash

**Reply by txducker** [reply]
Thank you for writing this software. I use it to combine pdf files after creating them from my scanner. The interface is easy to use and has been stable on Ubuntu 22.04.
Great job and very much appreciated!
Chad Chenault

**Comment by BH788** [comment]
Thanks development team, a great piece of software with a bunch of useful features that runs reliably and quickly. Keep up the good work.

**Comment by dreua** [comment]
I'm always glad to hear and read about it, if our work here was helpful to you, so thanks everyone for taking the time to write that down :blush: 

[I just changed the title of this discussion, unlike in issues Github doesn't show the original name or that I took that action.]

**Reply by dreua** [reply]
Btw: There also occasional shout-outs on [Mastodon](https://fosstodon.org/@pdfarranger) and recently I got one privately via email.

**Comment by cedman247** [comment]
Awesome awesome tool bridging a huge gap in a Windows environment. Was able to split and re-merge documents that were causing problems printing due to different page sizes in the pdf file

Many thanks.

**Comment by Moker05** [comment]
Hi devs, 
i was working on a pdf documents of about 160 pages, no software could do the job needed. then i found this jewel.
Thank You

**Comment by SimonEast** [comment]
Yes thank you team! I was getting fed up with other tools which were clunky and difficult. I just wanted to combine multiple PDFs by dragging them in, reorder the pages, delete some, rotate some, and save the new version. I was picturing a simple uncluttered window that could do this.

Google suggested your app (well third in the list) and I'm so glad I found it. (The other two were way too cumbersome!)

<img width="500" alt="Screenshot 00103" src="https://github.com/user-attachments/assets/3a7bf0fa-08b9-455d-8ef8-b1d8b7af069d" />

PDF Arranger will be my "go to" PDF manipulation app now. Thanks so much! Very appreciative. 😊
